src/web/components/Item/Item.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Item from './';
+
+describe('Item', () => {
+  let wrapper;
+  let data;
+  let content;
+
+  beforeEach(() => {
+    data = { id: 'GDJETI12K3J4L', name: 'turtle' };
+    content = <div>{data.name}</div>;
+    wrapper = shallow(<Item content={content} />);
```

Comment:
Why not to put the content as children instead ?

Reply:
I thought it was shorter to write (avoid closing the tag).
However, I can change it